frontend/src/components/leaderboard/ChildMedalComponent.vue

```diff
@@ -9,11 +12,10 @@ defineProps<{
   <div>
     <router-link to="/profile">
       <div>
-        <img
-          class="gilohm"
-          src="@/assets/sexy-guy-001-modified.png"
-          width="100"
-          height="100"
+        <RoundPictureComponent
+          :picture="ImportedPicture"
+          size="100px"
+          border-color="transparent"
         />
```

Comment:
Add class gilohm to move picture back in the middle
```suggestion
        <RoundPictureComponent
          class="gilohm"
          :picture="ImportedPicture"
          size="100px"
          border-color="transparent"
        />
```